File: server/src/main/java/com/backend/server/dto/BookingResponse.java
package com.backend.server.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class BookingResponse {

    private Long id;
    private Long spaceId;
    private Long userId;
    private LocalDateTime slotStart;
    private LocalDateTime slotEnd;
    private BigDecimal totalAmount;
    private String paymentStatus;
    private LocalDateTime createdAt;
}

File: app/src/main/java/com/example/client/data/model/BookingResponse.kt
package com.example.client.data.model

import com.google.gson.annotations.SerializedName

data class BookingResponse(
    @SerializedName("id") val id: Long,
    @SerializedName("spaceId") val spaceId: Long,
    @SerializedName("userId") val userId: Long,
    @SerializedName("slotStart") val slotStart: String,
    @SerializedName("slotEnd") val slotEnd: String,
    @SerializedName("totalAmount") val totalAmount: Double,
    @SerializedName("paymentStatus") val paymentStatus: String,
    @SerializedName("createdAt") val createdAt: String? = null
)
